RoutesApp_ios/modules/routelist/views/controllers/RouteListViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  RouteListViewController.swift
+//  RoutesApp_ios
+//
+//  Created by adri on 8/22/22.
+//
+
+import UIKit
+
+class RouteListViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    @IBOutlet weak var routeListTableView: UITableView!
+    var routeListDetailViewModel = RouteDetailViewModel()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupNavigationBar()
+        setIcon()
+        routeListDetailViewModel.getLines {
+            self.routeListTableView.reloadData()
+        }
+
+        routeListTableView.register(UINib.init(nibName: "RouteListTableViewCell", bundle: nil), forCellReuseIdentifier: "RouteListTableViewCell")
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        self.routeListTableView.reloadData()
+    }
+    
+    func setupNavigationBar() {
+        navigationItem.title = "Route List"
+        self.navigationItem.largeTitleDisplayMode = .always
+        navigationController?.navigationBar.barTintColor = UIColor(named: ConstantVariables.primaryColor)
+        let routeListVC = RouteListViewController()
+        let searchController = UISearchController(searchResultsController: routeListVC)
+        navigationItem.searchController = searchController
+        let colorValue = [NSAttributedString.Key.foregroundColor: UIColor.white]
+        UITextField.appearance(whenContainedInInstancesOf: [UISearchBar.self]).defaultTextAttributes = colorValue
+        UINavigationBar.appearance().isTranslucent = false
+    }
+    
+    func setIcon() {
+        let filterIcon = UIImage(named: "filter-icon")?.withRenderingMode(.alwaysOriginal)
+        let filterButton = UIBarButtonItem(image: filterIcon, style: .plain, target: self, action: #selector(showFilterPopUp))
+        
+        navigationItem.rightBarButtonItem = filterButton
+    }
+    
+    @objc func showFilterPopUp() {
+        
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return routeListDetailViewModel.routeListDetailModels.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let tableViewCell = tableView.dequeueReusableCell(withIdentifier: "RouteListTableViewCell", for: indexPath) as? RouteListTableViewCell else {
```

Comment:
same here, use a constant for the cell identifier

Reply:
Changed :)